sourcecode/apis/contentauthor/app/Console/Libraries/CliH5pFramework.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Console\Libraries;
+
+use App;
+use App\Libraries\ContentAuthorStorage;
+use App\Libraries\H5P\Framework;
+use GuzzleHttp\Client;
+use H5PPermission;
+use Illuminate\Support\Facades\DB;
+
+/**
+ * Override permission functions to enable CLI use
+ */
+class CliH5pFramework extends Framework
+{
+    public function __construct()
+    {
+        $contentAuthorStorage = app(ContentAuthorStorage::class);
+
+        parent::__construct(
+            new Client(),
+            DB::connection()->getPdo(),
+            $contentAuthorStorage->getH5pTmpDisk()
+        );
+    }
+
+    public function hasPermission($permission, $id = null): bool
```

Comment:
These methods should just return `true`, since the class is used exclusively in a CLI setting anyway. Anyone running commands should be presumed to have full permissions.